style.css

```diff
@@ -46,8 +47,76 @@ section {
   display: flex;
 }
 
+/* works section */
+
+.worksSection {
+  background-color: #fff;
+  min-height: 100vh;
+  display: grid;
+}
+
+.primaryCardText {
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  margin: 0;
+  line-height: 44px;
+  color: #091e42;
+}
+
+.seconndryCardText {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.mainListCategoris {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 0;
+  gap: 12px;
+}
+
+.mainTags {
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  padding: 8px 12px;
+  font-size: 14px;
+}
+
+.btn {
+  align-items: center;
+  gap: 10px;
+  background: var(--highlight);
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #ffff;
+  border-width: none;
+  border-style: none;
+  border-color: none;
+}
```

Comment:
- [ ] Good job adding button all along your web page 👏 , it could be better if your buttons  follow the button styling: Active, Hover and Pressed, it will increase the accessibility of your project 

| ![1](https://user-images.githubusercontent.com/80458199/234883996-598b9994-bf2b-4a59-83a5-945a049d6e18.PNG) | ![2](https://user-images.githubusercontent.com/80458199/234884041-efc05eaf-e85f-447d-8450-11495c3de5c0.PNG) | ![3](https://user-images.githubusercontent.com/80458199/234884079-a1358d47-3fef-453c-ad36-7909b14823c7.PNG) |